pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/SnapshotInternalStats.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.common.policies.data;
+
+public class SnapshotInternalStats {
```

Comment:
It is just a composite object that includes `ml name` and `ml internal stats`, why call it `SnapshotInternalStats`? Suggested giving it another name.

Reply:
How about changing the name to`SnapshotSystemTopicInternalStats`?

Reply:
I feel that this class is not necessarily bound to `Transaction snapshot`, we can call it `ManagedLedgerNameAndInternalStats`